hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/block/DeletedBlockLogImpl.java

```diff
@@ -130,9 +130,7 @@ public void incrementCount(List<Long> txIDs) throws IOException {
         DeletedBlocksTransaction block =
             scmMetadataStore.getDeletedBlocksTXTable().get(txID);
         if (block == null) {
-          // Should we make this an error ? How can we not find the deleted
-          // TXID?
-          LOG.warn("Deleted TXID {} not found.", txID);
+          LOG.debug("Deleted TXID {} not found.", txID);
```

Comment:
NIT: We can include the log inside check for debug log enabled.
```suggestion
          if (LOG.isDebugEnabled()) {
            LOG.debug("Deleted TXID {} not found.", txID);
          }
```